Dockerfile

```diff
@@ -4,7 +4,25 @@ FROM varnish:${VERSION}
 
 USER root
 
-RUN apt-get update && apt-get install -y prometheus-varnish-exporter &&\
- rm -rf /var/lib/apt/lists/*
+RUN set -e; \
+    apt-get update; \
+    apt-get -y install prometheus-varnish-exporter libmaxminddb-dev $VMOD_DEPS /pkgs/*.deb;
+
+# Install geoip2 https://github.com/varnishcache-friends/libvmod-geoip2
+RUN mkdir /tmp/module_to_build && \
+    curl -fLo /tmp/module_to_build/src.tar.gz https://github.com/varnishcache-friends/libvmod-geoip2/archive/refs/tags/v1.3.0.tar.gz
+
+WORKDIR /tmp/module_to_build
```

Comment:
not sure changing the workdir is a good idea

Reply:
probably it is not, I'm going to replace it by `cd ...`